### ISSUE TITLE
Scryfall-like links on card pages

### ISSUE THREAD

**Issue by RKelln** [dekkerglen/CubeCobra#743]
**What would you like to be able to do?**
I would like links to other useful resources and information about a particular card similar to the links on a cards scryfall page.

**Additional context**
![Screenshot from 2019-12-12 14-37-42](https://user-images.githubusercontent.com/63819/70743070-f9fa2000-1cec-11ea-8707-67cbbe335d2f.png)

**Comment by ArthurSilveira** [comment]
Working on this

**Comment by ArthurSilveira** [comment]
@RKelln What is the `card page`? The modal?

**Comment by dekkerglen** [comment]
Yes, like: https://cubecobra.com/tool/card/02c532e8-21f1-4d6f-a931-1ea48a6aeea2
